playground/demo.js

```diff
@@ -1,12 +1,24 @@
 function demo({ Client, Query, Zone }) {
   const client = new Client({ zone: Zone.EU });
-  const query = new Query('c0604b71c273c1fb3ef13eb2adfa4452');
+  const query = new Query('ebcd012c7f7172c3e16d7f40fe8a8842');
```

Comment:
If we change the hashid each time we use the demo for test this will be a nightmare. Traditionally we use this hashid: `6a96504dc173514cab1e0198af92e6e9`.